tracer/test/Datadog.Trace.Tests/Propagators/W3CTraceContextPropagatorTests.cs

```diff
@@ -178,40 +188,42 @@ public void TryParseTraceParent_Invalid(string header)
         }
 
         [Theory]
-        [InlineData("dd=s:2", 2, null, null)]
-        [InlineData("dd=o:rum", null, "rum", null)]
-        [InlineData("dd=t.dm:-4;t.usr.id:12345", null, null, "_dd.p.dm=-4,_dd.p.usr.id=12345")]
-        [InlineData("dd=s:2;o:rum;t.dm:-4;t.usr.id:12345~", 2, "rum", "_dd.p.dm=-4,_dd.p.usr.id=12345=")] // '~' is converted to '='
+        // valid
+        [InlineData("dd=s:2", 2, null, null, null)]                                                                                  // sampling priority
+        [InlineData("dd=s:-1", -1, null, null, null)]                                                                                // sampling priority
+        [InlineData("dd=o:rum", null, "rum", null, null)]                                                                            // origin
+        [InlineData("dd=t.dm:-4;t.usr.id:12345", null, null, "_dd.p.dm=-4,_dd.p.usr.id=12345", null)]                                // propagated tags
```

Comment:
I'm working forwards through this "part 3" PR and the "part 4" PR so I apologize if this is already done...but can you test tracestate substring `t.usr.id:baz64~~`? The resulting x-datadog-tags portion would be `_dd.p.usr.id=baz64==`

Reply:
Nevermind that is covered below 😄